org.webcamoid.Webcamoid.yml

```
@@ -0,0 +1,22 @@
+app-id: org.webcamoid.Webcamoid
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: webcamoid
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

with proper wayland support